x/evm/keeper/abci.go

```diff
@@ -11,7 +11,7 @@ import (
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 )
 
-// BeginBlock sets the block hash -> block height map for the previous block height
+// BeginBlock sets the block height -> header hash map for the previous block height
```

Comment:
if there's no hash -> height map how does `eth_getBlockByHash` work?

Reply:
I added it to the tendermint client so it's directly supported by the node 🙂

Reply:
good question btw